.github/dependabot.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+updates:
+
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+    allow:
```

Comment:
Should we also add `goja`? What was the criteria for add these in here?

Reply:
Yes, I was indecisive about Goja 😟 Although we have a direct dependency, k6 also has its own Gona. Any suggestions?

Reply:
I'd be happy top add it in and see how the bot behaves.